### PR TITLE
fix(security): Bump sys-mgmt-agent base image

### LINKED ISSUE #3740: sys-mgmt base image out-of-date
Bump system management agent Dockerfile from docker:20.10.7 to 20.10.8 to resolve base image vulnerabilities.

### PR DESCRIPTION
**Pull request by bnevis-i**
Closes #3740

Signed-off-by: Bryon Nevis <bryon.nevis@intel.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [X] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)  No functionality changed
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?) make run dev smoke test.
- [ ] I have opened a PR for the related docs change (if not, why?) Not applicable

## Testing Instructions
make docker; make run dev

## New Dependency Instructions (If applicable)

### CODE REVIEW

File: cmd/sys-mgmt-agent/Dockerfile [hunk 1]
```diff
@@ -31,7 +31,7 @@ RUN make cmd/sys-mgmt-agent/sys-mgmt-agent
 RUN make cmd/sys-mgmt-executor/sys-mgmt-executor
 
 # Get the Docker-in-Docker image layered-in now.
-FROM docker:20.10.7
+FROM docker:20.10.8
```
Comment (cloudxxx8) [inline cmd/sys-mgmt-agent/Dockerfile:34]: would it be better to use `docker:stable`?

Comment (bnevis-i) [reply cmd/sys-mgmt-agent/Dockerfile:34]: @lenny-intel  ?

Comment (bnevis-i) [reply cmd/sys-mgmt-agent/Dockerfile:34]: Getting really tired of bumping this container.

Comment (lenny-intel) [reply cmd/sys-mgmt-agent/Dockerfile:34]: I don't see a `stable` tag. Also typically we nee to pin it to an actual version.
https://hub.docker.com/_/docker?tab=tags&page=1&ordering=last_updated

**Review by lenny-intel — APPROVED** [review]
LGTM

File: cmd/sys-mgmt-agent/Dockerfile (hunk 1 above)
Comment (cloudxxx8) [reply cmd/sys-mgmt-agent/Dockerfile:34]: here is the stable tag, and it's old right now
https://hub.docker.com/layers/docker/library/docker/stable/images/sha256-ce46ea18798c578f8923e0ced4c9996c30363fc35edd1b1c40e5e542407818ac?context=explore

**Pull request merged**